data/human/weapons.txt

```diff
@@ -352,10 +352,10 @@ outfit "Mining Laser"
 		"reload" 1
 		"firing energy" 1.1
 		"firing heat" 2.1
-		"shield damage" .8
-		"hull damage" 1.4
-		"minable damage" 1.9
-		"prospecting" 1
+		"shield damage" .4
+		"hull damage" .7
+		"minable damage" .95
+		"prospecting" .5
```

Comment:
```suggestion
		"minable damage" 1.9
		"prospecting" 1
```
I dont see why we'd slow mining down (or if we want to do that it shouldnt be in the same PR, I'd argue)

keeping the old values makes this weapon make even more sense as a mining weapon

Reply:
Opted instead to reduce mineable HP by the same percentage damage was reduced by.